hack/collector-default

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------- #
+# Description                                                                      #
```

Comment:
nit: not sure how useful is the description since the comment and commands look self-explanatory.

Reply:
I'll remove the description block to avoid confusion.